timescale-forge/vpc-peering.md

```diff
@@ -0,0 +1,269 @@
+
+## Create a service with VPC attachment
+In the Timescale Forge console, navigate to the
+[Services Dashboard](https://console.forge.timescale.com/dashboard/services). Click
+`Create service` and select the compute and disk size which fits your needs.
+
+Expand the dropdown menu under the `Select a VPC` step and select the VPC you created
+previously. If you have multiple VPCs, select the VPC which you want your new service
+to be attached to.
+
+Click `Create Service`, and Timescale Forge will create your new service. Due to
+selecting a VPC during setup, your new service will be created with an attachment to
+your selected VPC.
+
+<img class="main-content__illustration" src="create-service-with-vpc.png" alt="Create new service with VPC attachment"/>
+
+## Migrating a service
+Timescale Forge services may be migrated between VPCs within a Forge project, and may also
+be migrated to and from the public network. Typically, once you have attached your service
+to a VPC, it should remain attached to ensure that your applications running in your AWS
+VPC will have continued connectivity to your service.
+
+<highlight type="warning">
+Timescale Forge uses a different DNS name for a Timescale service once it has been attached
+to a VPC. This means that you will need to update your connection string if migrating a service
+from the public internet into a VPC, or vice-versa.
+</highlight>
+
+To migrate a Timescale Forge service between VPCs, or to migrate to/from the public network,
+navigate to the operations page of your service. First, navigating to the
+[Services Dashboard](https://console.forge.timescale.com/dashboard/services), select the
+service you wish to migrate, click the `Operations` tab in the service details view, and
+click the `VPC` tab within the `Operations` view. From this view, you have a few options
+depending on the state of your service.
+
+<highlight type="warning">
+DNS propagation can take time. After migrating your Timescale Forge service,
+please allow a few minutes for DNS propagation. If you receive DNS errors indicating
+that the DNS name could not be resolved, this indicates that more time is needed for
+DNS propagation.
+</highlight>
+
+### Migrate from public network to VPC
+When your service is attached to the public network, you have the option to select a VPC
+to migrate your service into.
+
+<img class="main-content__illustration" src="migrate-public-to-vpc.png" alt="Migrate from public network to VPC"/>
+
+Once you have selected the VPC to migrate your service into, click `Attach VPC`. When prompted
+for confirmation, type `MIGRATE` as instructed, then click `Migrate`.
+
+<img class="main-content__illustration" src="migrate-public-to-vpc-confirm.png" alt="Confirm migration into VPC"/>
+
+Your service will now go through a series of asynchronous operations ultimately
+resulting in your service being attached to your selected VPC. These operations
```

Comment:
`your service being attached` -> `it attaching`

Reply:
I'm actually going to simplify this paragraph a bit. I think it is too verbose, definitely more than it needs to be.